Makefile

```diff
@@ -222,6 +222,20 @@ unsigned-ios: stop pre-build check-style ## Build an unsigned version of the iOS
 	@rm -rf build-ios/
 	@ps -ef | grep -i "cli.js start" | grep -iv grep | awk '{print $$2}' | xargs kill -9
 
+unsigned-ios-x86_64: stop pre-build check-style ## Build an unsigned x86_64 version of the iOS app for iPhone simulator
```

Comment:
wondering if the target should be renamed, something like `ios-sim-x86_64` also add the target to the .PHONY at the top of the Makefile